### PR TITLE
fix tinymce issues in collection context

### PR DESCRIPTION
**Pull request by itkg-nanne**
[OO-BUGFIX] fix tinymce issues in collection context

### CODE REVIEW

File: BackofficeBundle/Resources/public/coffee/addPrototype.coffee [hunk 1]
```diff
@@ -37,7 +37,7 @@ PO.formPrototype = (collectionHolder, settings) ->
     return
 
   @toogleAddButton()
-  @maxIndex = @getIndex()
+  @getNextIndex()
```
Comment (alavieille) [inline BackofficeBundle/Resources/public/coffee/addPrototype.coffee]: this call is used ?

File: BackofficeBundle/Resources/public/coffee/addPrototype.coffee [hunk 2]
```diff
@@ -47,6 +47,15 @@ PO.formPrototype:: =
   getIndex: ->
     return @collectionHolder.children('div').length
 
+  getNextIndex: ->
+  	index = 0
```
Comment (alavieille) [inline BackofficeBundle/Resources/public/coffee/addPrototype.coffee]: can you fix align

**Comment by itkg-nanne** [comment]
update done @alavieille

**Comment by alavieille** [comment]
:+1:

**Pull request merged**